medicines/docs/contributor_guidelines/README.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+## Contents of this file
+
+### For contributors
+
+- [Code of conduct](#code-of-conduct)
+
+## Code of Conduct
+
+Please read our [code of conduct](./code-of-conduct.md) before contributing.
```

Comment:
Is it deliberate that we have ./docs/snake_case/kebab-case.md? I'd expect kebab-case for the directories as well as the markdown files.

Reply:
Ended up inadvertently following the conventions of MoJ, where I drew some of this content from - fixed now!